vars/folioUI.groovy

```diff
@@ -129,9 +129,17 @@ void build(String okapiUrl, OkapiTenant tenant, boolean isEureka = false, String
         webOrigins                  : ["/*"],
         authorizationServicesEnabled: true,
         serviceAccountsEnabled      : true,
-        attributes                  : ['post.logout.redirect.uris': "/*##${tenantUrl}/*", login_theme: 'custom-theme']
+        attributes                  : ['post.logout.redirect.uris': "/*##${tenantUrl}/*", login_theme: 'custom-theme'],
+      ]
+      Map ssoUpdates = [
+        ssoSessionIdleTimeout       : 7200,
+        ssoSessionMaxLifespan       : 7200,
+        clientSessionIdleTimeout    : 7200,
+        clientSessionMaxLifespan    : 7200,
       ]
       client.put(updateRealmUrl, JsonOutput.toJson(updateContent), headers)
+      //RANCHER-1762 | MUST HAVE | Keycloak SSO session timeout settings
+      client.put("https://${keycloakDomain}/admin/realms/${tenantId}", JsonOutput.toJson(ssoUpdates), headers)
```

Comment:
Let's start deprecation of JsonOutput and JsonSlurper. Let's use readJSON and writeJSON